packages/block-library/src/section/editor.scss

```diff
@@ -0,0 +1,61 @@
+/**
+ * Section: All Alignment Settings
+ */
+.wp-block[data-type="core/section"] {
+	@include break-small() {
+		padding-left: $block-padding*3;
```

Comment:
This rule is what's causing the horizontal scrollbar. If you remove this entire rule (lines 5-8) everything seems to work fine. Can you recall why this was added in the first place?

Reply:
I believe I've fixed this as of

https://github.com/WordPress/gutenberg/pull/13964/commits/696a15bd91e81dce0d83465f1a32e7b8f29d3d6a

I don't know why that rule was added as I've been diligent about comments on the other lines to ensure it's clear. Removing and adding compensating margins seemd to do the trick and avoid the 1px overflow bug.

I can no longer replicate in any browser 🤞 

cc @kjellr